app/src/mixin.js

```diff
@@ -0,0 +1,10 @@
+let mixin = {
+    methods: {
+        checkLoggedIn: function(error) {
+            return error.includes('Not logged in') ?
+                true : false
+        }
+    }
+}
+
```

Comment:
Very cool

Reply:
side note, does node have any kind of solution for passing an error code to the front end with the error?  this works for me tho lol